### PR TITLE
updated the note

### PR DESCRIPTION
**Pull request by MaratMussabekov**
fixes https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10508

### CODE REVIEW

File: windows/security/information-protection/kernel-dma-protection-for-thunderbolt.md [hunk 1]
```diff
@@ -92,7 +92,10 @@ Beginning with Windows 10 version 1809, you can use the Windows Security app to
    - Reboot system into Windows.
 
    >[!NOTE]
-   > **Hyper-V - Virtualization Enabled in Firmware** is not available when **A hypervisor has been detected. Features required for Hyper-V will not be displayed.** is displayed. This means that **Hyper-V - Virtualization Enabled in Firmware** is set to Yes and the **Hyper-V** Windows feature is enabled. Enabling Hyper-V virtualization in Firmware (IOMMU) is required to enable **Kernel DMA Protection**, even when the firmware has the flag of "ACPI Kernel DMA Protection Indicators" described in [Kernel DMA Protection (Memory Access Protection) for OEMs](/windows-hardware/design/device-experiences/oem-kernel-dma-protection).
+   > If the **Hyper-V** Windows feature is enabled, all the Hyper-V-related features will be hidden, and **A hypervisor has been detected. Features required for Hyper-V will not be displayed** entity will be shown on the bottom of the list. It means that **Hyper-V - Virtualization Enabled in Firmware** is set to YES.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/kernel-dma-protection-for-thunderbolt.md:95]:
```suggestion
   > If the **Hyper-V** Windows feature is enabled, all the Hyper-V-related features will be hidden, and **A hypervisor has been detected. Features required for Hyper-V will not be displayed** entity will be shown at the bottom of the list. It means that **Hyper-V - Virtualization Enabled in Firmware** is set to YES.
```

File: windows/security/information-protection/kernel-dma-protection-for-thunderbolt.md [hunk 2]
```diff
@@ -92,7 +92,10 @@ Beginning with Windows 10 version 1809, you can use the Windows Security app to
...
+
+   >[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/kernel-dma-protection-for-thunderbolt.md:97]:
```suggestion
   > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by SillyKeith** [comment]
Are there 3 Notes now?   I think this is more confusing. 

Also, perhaps the feature team can get you a better MSINFO32 screenshot?  One that zooms into just the area of interest and without all the MS internal Build#s and codenames (Palindrome in System Mfgr).  I've attached an example (Don't use mine though since my system is missing a security feature). A Surface PC might be a better one to take it from... 

MSINFO32

File: windows/security/information-protection/kernel-dma-protection-for-thunderbolt.md [hunk 3]
```diff
@@ -92,7 +92,10 @@ Beginning with Windows 10 version 1809, you can use the Windows Security app to
    - Reboot system into Windows.
 
    >[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/kernel-dma-protection-for-thunderbolt.md:94]:
```suggestion
   > [!NOTE]
```

**Comment by MaratMussabekov** [comment]
Hello,
Are there 3 Notes now? I think this is more confusing. - no, only two. I separated the part where it said that IOMMU is required.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — APPROVED** [review]

**Comment by aczechowski** [comment]
Thanks @MaratMussabekov for the contribution.
@SillyKeith if things still aren't clear based on your original feedback #10508, please file additional feedback. Also, regarding the screenshot, probably best to file a separate issue specific to that request.

**Pull request merged**